src/users/index.ts

```diff
@@ -0,0 +1,65 @@
+import jsonwebtoken from "jsonwebtoken"
+import { requestGet, requestPost, requestDelete } from "../entityRequest"
```

Comment:
Lint, which is run on publish,  is complaining about the order of this being alphabetized- requestDelete should be first.